lte/gateway/fabfile.py

```diff
@@ -855,7 +911,7 @@ def _run_integ_tests(gateway_ip='192.168.60.142', tests=None, federated_mode=Fal
     host = env.hosts[0].split(':')[0]
     port = env.hosts[0].split(':')[1]
     key = env.key_filename
-    tests = tests or ''
+    tests = "TESTS=" + ",".join(tests) if tests else ''
```

Comment:
This does not work TESTS=.... only seems to be able to handle one testcasse file

Reply:
The tests list should be separated by space here and not comma.

Even with the fix to use space as delimiter here, it will not work as it needs few more changes. Like 
- the multiple test-scripts names list should be sent under quotes as argument, for the Make file to take it as a single argument, e.g., `make integ_test TESTS="<testcase1> <testcase2>".`  (in line number 941)
- The magma/lte/gateway/python/integ_tests/Makefile needs to be modified to generate the result xml file as a common filename when multiple testscript names are provided in argument `TESTS=`, e.g.
`timeout --foreground -k 930s 900s sudo -E PATH=$(PATH) PYTHONPATH=$(PYTHONPATH):$(S1AP_TESTER_PYTHON_PATH) $(BIN)/pytest $(FLAKY_CMD_ARGS) --capture=tee-sys --junit-xml=$(RESULTS_DIR)/$(basename $(notdir test_result_with_multiple_testcases)).xml -x $(1) || (echo "fail" > $(MAGMA_ROOT)/test_status.txt && exit 1)`

Reply:
This does not seem to work with space seperated tests as well. There seems to be a foreach loop missing here https://github.com/magma/magma/blob/master/lte/gateway/python/integ_tests/Makefile#L52. I will drop this feature for now.